tests/network/conftest.py

```diff
@@ -231,3 +221,20 @@ def cnao_deployment(hco_namespace):
         namespace_name=hco_namespace.name,
         deployment_name=CLUSTER_NETWORK_ADDONS_OPERATOR,
     )
+
+
+@pytest.fixture(scope="session", autouse=True)
+def network_sanity(hosts_common_available_ports, junitxml_plugin):
+    """
+    Perform verification that the cluster is a multi-nic one otherwise exit run
+    """
+    network_sanity_failure_return_code = 91
```

Comment:
Can you please explain what is this error value (91)?

Reply:
may be it worth to add a short comment line?

Reply:
exit_pytest_execution() already explains it.

Reply:
> exit_pytest_execution() already explains it.

I went over the function but didn't find an explanation for this return code.
And anyway - this code is explicitly used here, so I think this is the right place to explain its meaning.
Actually, I think that the more preferred way would be to store this number in a constant with a meaningful name (even if it is used once), otherwise it looks like a magic number.